content/doc/book/using/using-nodes.adoc

```diff
@@ -0,0 +1,426 @@
++
+[source,bash]
+----
+ubuntu@DESKTOP-LUFH4P5:~$ ssh-keygen
+Generating public/private rsa key pair.
+Enter file in which to save the key (/home/ubuntu/.ssh/id_rsa):
+Enter passphrase (empty for no passphrase):
+Enter same passphrase again:
+Your identification has been saved in /home/ubuntu/.ssh/id_rsa
+Your public key has been saved in /home/ubuntu/.ssh/id_rsa.pub
+The key fingerprint is:
+SHA256:XqxxjqsLlvDD0ZHm9Y2iR7zC6IbsUlMEHo3ffy8TzGs ubuntu@DESKTOP-LUFH4P5
+The key's randomart image is:
++---[RSA 3072]----+
+|  o+             |
+| ...o  .         |
+|  .o .+ .        |
+|    o+.+ o o     |
+|  ... o.So* .    |
+|  o+ = +.X=      |
+| o oO + *..+     |
+|. oo.o o .E .    |
+| o... oo.. o     |
++----[SHA256]-----+
+ubuntu@DESKTOP-LUFH4P5:$
+----
+
+==== Registering your SSH key at jenkins Credentials
+
+1. Go to your jenkins dashboard; 
+2. Go to `Manage Jenkins` option in main menu;
++++
+<span>
+  <button 
+    data-target="#credentials-1" 
+    aria-controls="credentials-1"
+    aria-expanded="false" class="btn btn-secondary btn-sm" type="button" data-toggle="collapse" 
+    >
+    <svg class="bi bi-exclamation-circle-fill" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+        <path fill-rule="evenodd" d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 4a.905.905 0 0 0-.9.995l.35 3.507a.552.552 0 0 0 1.1 0l.35-3.507A.905.905 0 0 0 8 4zm.002 6a1 1 0 1 0 0 2 1 1 0 0 0 0-2z"/>
+    </svg>
+    Info
+  </button>
+</span>
+<div class="collapse" id="credentials-1">
+  <div class="card card-body my-3">
++++
+    image:node/credentials-1.png[credentials menu]
++++
+  </div>
+</div>
++++
+3. select the drop option `add Credentials` from the global item;
++++
+<span>
+  <button 
+    data-target="#credentials-2" 
+    aria-controls="credentials-2"
+    aria-expanded="false" class="btn btn-secondary btn-sm" type="button" data-toggle="collapse" 
+    >
+    <svg class="bi bi-exclamation-circle-fill" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+        <path fill-rule="evenodd" d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 4a.905.905 0 0 0-.9.995l.35 3.507a.552.552 0 0 0 1.1 0l.35-3.507A.905.905 0 0 0 8 4zm.002 6a1 1 0 1 0 0 2 1 1 0 0 0 0-2z"/>
+    </svg>
+    Info
+  </button>
+</span>
+<div class="collapse" id="credentials-2">
+  <div class="card card-body my-3">
++++
+    image:node/credentials-2.png[add credentials option]
++++
+  </div>
+</div>
++++
+
+4. Fill the form:
+** Kind:  SSH Username with private key;
+** id: jenkins
+** description: The jenkins ssh key
+** username: jenkins
+** Private Key: select `Enter directly` and press the Add button to insert yout private key;
++
+[NOTE]
+====
+If you generate it at the default location use the command to print it on terminal: `cat ~/.ssh/id_rsa`
+====
+** Passphrase: fill your passphrase used to generate the SSH keys and then press OK.
++++
```

Comment:
Please avoid RAW HTML

Reply:
I understand your point. But the asciidoctor and awestruct don't offer us an option to use toggle visualization or tabs views, so I use the bootstrap already used by jenkinks.io.

I already open issue #3480 To try to improve our plugins, but I've no experience with Ruby programming. have you any expertise with to help me? 

But about the image, What do you prefer? 
 - Remove the HTML and don't use the toggle visualization?
- O keep it to improve user experience?